### DISCUSSION TITLE
Would you mind to start a gitter.im community to ease the exchange with other users and dev team?

### DISCUSSION THREAD

**Discussion by swingstate** [photoprism/photoprism#770] [Ideas]
I don't want to create an issue for every question and I think gitter is much more user friendly than this forum style? Hence, my question regarding gitter.im :)

What do you think @lastzero?

**Comment by lastzero** [comment]
We already have a community chat there 🌈 check our badges in README.md

**Comment by swingstate** [comment]
no clue :) The readme does not show any link to a chat or maybe I am blind today.

**Reply by lastzero** [reply]
![Screenshot_20201229_123824](https://user-images.githubusercontent.com/301686/103281294-c317aa80-49d2-11eb-949e-275df3c73214.jpg)

**Comment by swingstate** [comment]
... my old eyes :) I am still not 100% familiar with the git ecosystem. Thanks!
